Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,28 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int personCount = 2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'personCount'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5050127)

Reply:
Added const modifier to latest commit